app/javascript/src/components/Dashboard/Notes/NewNoteForm.jsx

```diff
@@ -1,58 +1,95 @@
-import React from "react";
+import React, { useRef } from "react";
 
+import { Input, Textarea, Select } from "@bigbinary/neetoui/v2";
 import { Formik, Form } from "formik";
-import { Button } from "neetoui";
-import { Input, Textarea } from "neetoui/formik";
 import * as yup from "yup";
 
-import notesApi from "apis/notes";
+import Panel from "./Panel";
 
-export default function NewNoteForm({ onClose, refetch }) {
-  const handleSubmit = async values => {
-    try {
-      await notesApi.create(values);
-      refetch();
-      onClose();
-    } catch (err) {
-      logger.error(err);
+const TAGS = ["Getting Started", "Onboarding", "User Flow", "UX", "Bugs", "V2"];
+
+const TAGS_OPTIONS = TAGS.map(val => {
+  return {
+    label: val,
+    value: val.toLowerCase().split(" ").join("_")
+  };
+});
+
+const NewNoteForm = ({ showPane, onClose }) => {
+  const formRef = useRef();
+
+  const handleSubmit = () => {
+    //handle form submit
+  };
+
+  const handleForm = () => {
+    if (formRef.current) {
+      formRef.current.handleSubmit();
     }
   };
+
   return (
-    <Formik
-      initialValues={{
-        title: "",
-        description: ""
-      }}
-      onSubmit={handleSubmit}
-      validationSchema={yup.object({
-        title: yup.string().required("Title is required"),
-        description: yup.string().required("Description is required")
-      })}
+    <Panel
+      title="Add New Note"
+      showPane={showPane}
+      onClose={onClose}
+      onSubmit={() => handleForm()}
     >
-      {({ isSubmitting }) => (
-        <Form>
-          <Input label="Title" name="title" className="mb-6" />
-          <Textarea label="Description" name="description" rows={8} />
-          <div className="nui-pane__footer nui-pane__footer--absolute">
-            <Button
-              onClick={onClose}
-              label="Cancel"
-              size="large"
-              style="secondary"
-            />
-
-            <Button
-              type="submit"
-              label="Submit"
-              size="large"
-              style="primary"
-              className="ml-2"
-              disabled={isSubmitting}
-              loading={isSubmitting}
-            />
-          </div>
+      <Formik
+        innerRef={formRef}
+        initialValues={{
+          title: "",
+          description: "",
+          assignedContact: "",
+          tags: ""
+        }}
+        onSubmit={handleSubmit}
+        validationSchema={yup.object({
+          title: yup.string().required("Title is required"),
+          description: yup.string().required("Description is required"),
+          assignedContact: yup
+            .string()
+            .required("Assigned Contact is required"),
+          tags: yup.string().required("Tag is required")
+        })}
```

Comment:
Please move this to a constants file in the scope of the component along with `yup` import.